### ISSUE TITLE
screen width smaller

### ISSUE THREAD

**Issue by arthurmelton** [Mafrans/StadiaPlus#132]
if I have my screen width smaller the images get shrinked
![image](https://user-images.githubusercontent.com/29708070/98129064-ad7c7d80-1e86-11eb-8886-9e5cfca1693c.png)
you could make it where say it shows 2 games per row

**Comment by ELowry** [comment]
My typical approach to this would be one of the following:
```css
.stadiaplus_libraryfilter .stadiaplus_libraryfilter-gamecontainer .stadiaplus_libraryfilter-game {
    width: calc(25% - 1.5rem);
    height: 32vh;
    max-height: 20vw;
}
```
or
```css
.stadiaplus_libraryfilter .stadiaplus_libraryfilter-gamecontainer .stadiaplus_libraryfilter-game {
    width: calc(25% - 1.5rem);
    height: 20vw;
    max-height: 32vh;
}
```

This way the aspect ratio gets clamped beyond a certain point.

**Comment by ELowry** [comment]
This has been addressed in Pull Request #140 .

**Comment by ELowry** [comment]
If pull request #140 causes conflicts/issues, I would be happy to split out only the relevant sections.
